solidity/contracts/ViewBTC.sol

```diff
@@ -365,10 +378,17 @@ library ViewBTC {
     /// @notice         calculates the Proof of Work hash of the header
     /// @param _header  the header
     /// @return         the Proof of Work hash
-    function work(bytes29 _header) internal view typeAssert(_header, BTCTypes.Header) returns (bytes32) {
+    function workHash(bytes29 _header) internal view typeAssert(_header, BTCTypes.Header) returns (bytes32) {
         return _header.hash256();
     }
 
+    /// @notice         calculates the Proof of Work hash of the header, and converts to an integer
+    /// @param _header  the header
+    /// @return         the Proof of Work hash
```

Comment:
A little nit-picky, but maybe `the Proof of Work hash as an integer` just to be clear?